superset-frontend/src/explore/components/controls/FilterControl/AdhocFilterPopoverTrigger/AdhocFilterPopoverTrigger.test.tsx

```diff
@@ -73,7 +73,10 @@ test('should be visible when controlled', async () => {
       Click
     </AdhocFilterPopoverTrigger>,
   );
-  expect(screen.getByRole('tooltip')).toBeInTheDocument();
+
+  await waitFor(() => {
```

Comment:
I think we can replace `waitFor + getByRole` with `findByRole`

Reply:
Agreed, thanks!